package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vrkkpa-samlp",
-  "version": "3.4.3-sf1.1",
+  "version": "4.0.0-sf2.0",
```

Comment:
This should probably be `4.0.0-sf1.0` instead of sf2.0 since it is first sf version of major 4.0.0 of upstream